src/models/BaseModel.py

```diff
@@ -75,12 +88,15 @@ def __init__(self, train_size=0.75, test_size=None, force_load_all=False):
                                                            train_size=train_size,
                                                            test_size=test_size,
                                                            random_state=1)
-
-        self.description_train_data, self.description_test_data = train_test_split(
-            self.description_feature,
-            train_size=train_size,
-            test_size=test_size,
-            random_state=1)
+        try:
+            self.description_train_data, self.description_test_data = train_test_split(
+                self.description_feature,
+                train_size=train_size,
+                test_size=test_size,
+                random_state=1)
+        except:
```

Comment:
yeah for me it depends, in this case i don't think it's an intentional flow and i'd rather handle it with a try/except. If it was an intentional, well defined, flow i would have taken an if